.circleci/config.yml

```diff
@@ -248,25 +239,10 @@ jobs:
           command: |
             python -m scripts.run_e2e_tests --skip-install --skip-build --suite="profileFeatures" --prod_env
 
-  e2e_accessibility_and_embedding:
```

Comment:
Moving tests back to TravisCI in an effort to load balance, but I've still maintained test grouping so that we can avoid the problem of rebuilding.